### PR TITLE
Airflow in Jenkins CI

### PR DESCRIPTION
**Pull request by backstreetkiwi**
## Description
Added the files necessary to execute the Airflow integration tests in Jenkins CI.
A test run was executed on this branch, you find the results here: https://ci.stackable.tech/view/02%20Component%20Tests%20(flex)/job/Airflow%20Operator%20Integration%20Tests%20(flex)/5/

After this PR is merged, we can add a nightly Job for Airflow running on `main` branch against the latest nightly binary. 

## Review Checklist
- [x] Code contains useful comments
- [x] Changelog updated (or not applicable)

### CODE REVIEW

File: airflow-operator/.ci/integration-tests/ionos-centos-8/test.sh [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+" > set-up-redis.sh
+scp set-up-redis.sh testdriver-1:set-up-redis.sh
+ssh testdriver-1 chmod a+x set-up-redis.sh
+ssh testdriver-1 ./set-up-redis.sh
+
+# Wait for Redis to be up and running
+echo Starting Redis ...
+while [ "$(kubectl get pod airflow-redis-replicas-2 --output=jsonpath='{.status.containerStatuses[0].ready}')" != "true" ]; do
+	sleep 2
+done
```
Comment (siegfriedweber) [inline airflow-operator/.ci/integration-tests/ionos-centos-8/test.sh:36]:
```suggestion
while [ "$(kubectl get statefulset redis-master --output=jsonpath='{.status.readyReplicas}')" != "1" ] || 
      [ "$(kubectl get statefulset redis-replicas --output=jsonpath='{.status.readyReplicas}')" != "3" ]; do
    sleep 2
done
```

**Review by siegfriedweber — APPROVED** [review]
LGTM

There is only one small suggestion for the test setup script.

File: airflow-operator/.ci/integration-tests/ionos-centos-8/test.sh (hunk 1 above)
Comment (backstreetkiwi) [reply airflow-operator/.ci/integration-tests/ionos-centos-8/test.sh:36]: @siegfriedweber Good idea, it is more exact on what we really expect. Are you sure the StatefulSet's name ist `redis-master`, not `airflow-redis-master` ?

Comment (backstreetkiwi) [reply airflow-operator/.ci/integration-tests/ionos-centos-8/test.sh:36]: I started a local run to find out...

Comment (backstreetkiwi) [reply airflow-operator/.ci/integration-tests/ionos-centos-8/test.sh]: It has to be `airflow-redis-master`, so I changed it to that.

**Comment by adwk67** [comment]
LGTM(too!)

**Comment by backstreetkiwi** [comment]
Test runs successfully after the change recommended by @siegfriedweber

**Review by adwk67 — APPROVED** [review]
LGTM

**Pull request merged**